src/accelerate/data_loader.py

```diff
@@ -38,6 +38,12 @@
 if is_tpu_available(check_device=False):
     import torch_xla.distributed.parallel_loader as xpl
 
+    class MpDeviceLoaderWrapper(xpl.MpDeviceLoader):
+        @property
+        def total_batch_size(self):
+            return self._loader.total_batch_size
```

Comment:
Let's add a docstring to the class here stating briefly what it is and the new property, mimicking how [Accelerator](https://github.com/huggingface/accelerate/blob/docs-revamp/src/accelerate/accelerator.py#L140-L152) has it, since this will be useful for folks and it should show in the documentation as it's different from the base torch dataloader.

Similarly add this property doc to the other two classes as well.